spot-contracts/contracts/PerpetualTranche.sol

```diff
@@ -593,30 +532,20 @@ contract PerpetualTranche is
             type(uint256).max
         );
 
-        // verifies if rollover amount is acceptable
-        if (r.trancheInAmt == 0 || r.tokenOutAmt == 0 || r.perpRolloverAmt == 0) {
+        // Verifies if rollover amount is acceptable
+        if (r.trancheInAmt == 0) {
```

Comment:
Is there motivation for removing the `tokenOutAmt` check?

Reply:
From the spot contract perspective, we care more about tokens coming it. 

ie) trancheInAmt > 0 and tokenOutAmt >= 0 is reasonable. However trancheInAmt = 0 is unreasonable.

Reply:
On second thought, adding it back for the sake of consistency. In other functions like deposit we check for either the note == 0 or the underlyingIn == 0 to revert.

Reply:
Makes sense